### DISCUSSION TITLE
How can I find the Sync version of a new release?

### DISCUSSION THREAD

**Discussion by LorenAmelang** [TriliumNext/Trilium#2623] [Q&A] [answered]
I finally have a server working, but as described in 
https://github.com/zadam/trilium/discussions/2598
the install is a _long_ process. I seem to have Sync version: 23 ...

So if I update my client, how do I know if it will break the server connection? 

Sounds like v0.50.1 might well break the server connection:

https://github.com/zadam/trilium/discussions/2609
optimized sync protocol which can now in some cases halve the sync/time

Is there some place to see the Sync version for new releases - without installing them?

**Comment by zadam** [comment, marked as answer]
If you are really interested, you can pull that info from the history of https://github.com/zadam/trilium/blob/master/src/services/app_info.js

It's not documented anywhere. The rule of thumb is that you should use same minor version (so e.g. 0.50.X) on all sync instances. There are exceptions (sync version changed in patch release), but that would be visible in the release notes.

In your case 0.50.X can be synced only with other 0.50.X instances.

**Reply by LorenAmelang** [reply]
Thanks! Now I know where to look...  

So I suppose the process is to quit all clients and stop the server. Update the server, verify it starts. Maybe try it with web access? Then update sync clients...  Does it matter what order they are restarted in? 

If there is a sync version conflict, what happens? Just an error message? Or a possibility of data corruption?

**Reply by zadam** [reply]
>  Does it matter what order they are restarted in?

No.

> If there is a sync version conflict, what happens? Just an error message? Or a possibility of data corruption?

There will be an error message when attempting to sync. No data corruption should occur.
